pom.xml

```diff
@@ -35,14 +35,6 @@
             <version>1.5.2.RELEASE</version>
             <scope>compile</scope>
         </dependency>
-        <dependency>
```

Comment:
Is there a reason that you removed lombok?

Reply:
I removed it for multiple reasons:
- I perceive it as impractical to go through a write -> generate code -> use function development cycle. It is slow and cumbersome
- The lib adds complexity (and resources) to every single build to replace something that can be generated by your IDE once in a matter of seconds
- it was only used in one place and not all the other places it could have been used.

If you insist on it I'll add it back and try to also refactor the other places where it could be used. I just don't see the point of it (in this library at least, in larger projects there might be a really good use for it).

Reply:
Fair enough points. Let's stick to the version without Lombok. This also makes it easier for contributors to join.